routes/projects.js

```diff
@@ -653,99 +247,94 @@ router.route('/:id') //This is where the classifier would be
                         }, function(err, users) {
+                            if (err) { return renderError(res, err); }
                             userFinal = users;
                             // logger.info(userFinal);//This works!!!
                         });
                         mongoose.model("neume").find({
                             project: project._id
                         }, function(err, neumes) {
-                            neumes.forEach(function(neume) { //Change this to a for loop to make the data faster. Right now the performance is almost 5 minutes.
-
-                                if(neume.classifier == undefined){
-                                    logger.info("no classifier")
-                                }
-
-                              else if(neume.classifier.includes(".xlsx")){
-                                var image = neume.imageMedia;
-                                //logger.info(image);
-                                if (fs.existsSync('exports/xl/media/' + image)) {
-                                    var imgPath = 'exports/xl/media/' + image; //This is undefined.
-                                    var A = storedImages;
-                                    var a = new A;
-                                    a.projectID = project._id;
-                                    a.neumeID = neume._id;
-                                    a.img.data = fs.readFileSync(imgPath);
-                                    a.img.contentType = 'image/png';
-                                    a.imgBase64 = a.img.data.toString('base64');
-
-                                    imageData.push(a.img.data.toString('base64')); //This works for all the images stored in the database.
-                                    //logger.info(imageData); //This works
-                                    mongoose.model('neume').find({
-                                        _id: neume._id
-                                    }).update({
-                                            //push the neumes into the imagesBinary array
-                                            imagePath: 'exports/xl/media/' + neume.imageMedia,
-                                            imagesBinary: fs.readFileSync('exports/xl/media/' + neume.imageMedia).toString('base64')
-                                        },
-
-                                        function(err, data) {
-                                            //logger.info(err, data);
-                                            imageData = [];
-
-                                            a.save(function(err, a) {
-                                                if (err) throw err;
-
-                                                logger.error('saved img to mongo');
+                            if (err) { return renderError(res, err); }
+                            for (var i = 0; i < neumes.length; i++) {
+                                var neume = neumes[i];
```

Comment:
If `i` is only used to get the current neume, then this may be a better structure:
```suggestion
                            for (let neume of neumes) {
```